rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -3048,6 +3048,18 @@ public String loginWithCredential(@MultipartForm LoginForm multipart)
         return s.getMappedInfo("ProActiveScheduler:name=RuntimeData");
     }
 
+    @Override
+    @GET
+    @Path("stathistory")
+    @Produces("application/json")
+    public String getStatHistory(@HeaderParam("sessionid") String sessionId) throws NotConnectedRestException {
```

Comment:
Is `throws NotConnectedRestException` required ?

Reply:
yes